index.html

```diff
@@ -4,9 +4,99 @@
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Hello, YOUR_NAME</title>
+  <title>Hello, Kendahl</title>
+  <style>
+
+html {
+  background: linear-gradient(-45deg, #ee7752, #e73c7e, #23a6d5, #23d5ab);
+	background-size: 400% 400%;
+	animation: gradient 10s ease infinite;
+}
+
+@keyframes gradient {
+	0% {
+		background-position: 0% 50%;
+	}
+	50% {
+		background-position: 100% 50%;
+	}
+	100% {
+		background-position: 0% 50%;
+	}
+}
+
+  div.gallery {
+    margin: 5px;
+    border: 1px solid gold;
+    float: left;
+    width: 180px;
+    /* filter: grayscale(80%) */
+  }
+  div.gallery img {
+  width: 100%;
+  height: auto;
+  /* transform: skew(10deg) translate(150px);
+  transform-origin: bottom left; */
+}
+
+div.gallery:hover {
+  border: 3px solid purple;
+}
+
+  </style>
 </head>
 <body>
-  <!-- use this space to add your own content -->
+  <h1><center>Welcome to the Danga Zone!!!</center></h1>
+<ul>
+<li>Hey there, my name is Kendahl Ryan Ross and I'm a Davidson College Sophomore.</li>
+<li> I'm an intended Communications major and I'm taking this course to begin
+my epic journey to become a legendary coder.</li>
+<li>I have no previous background in coding and creating content on the web, but one day I'll be great at it.</li>
+<li>My goals are to be able to read/write/communicate code in a professional
+  environment and have my own personal web page up and running soon.</li>
+  <li><a href="./index2.html"> Have a peek behind the curtain.</a></li>
+</ul>
+<h2><u>Projects I find interesting</h2></u>
```

Comment:
1. Use `text-decoration` if you want to make some text underlined.
2. Tags can't overlap like this; you opened `<u>` inside `<h2>`, you have to close it before you close `<h2>`. The HTML validator would catch this.

When a browser sees Invalid HTML it manipulates the HTML until it's valid. There are no rules about how a browser "should" repair invalid HTML, so you often wind up rendering HTML that's pretty different from what you intended. If you don't realize that's what's happening, you introduce little hacks to get the page to "look right" when valid HTML would've done what you expected the whole time.

So, double check!